src/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ set_target_properties(gpuarray PROPERTIES
   INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib
   MACOSX_RPATH OFF
   # This is the shared library version
-  VERSION 0.0
+  VERSION 0.1
```

Comment:
I'll come back to the versioning. Now have a new interface. This won't trigger the recompilation and won't give good user warning. If people update Theano but not libgpuarray, then they will get compilation error related to the convolution as GA_CTX_PROP_LARGEST_MEMBLOCK isn't defined. Check jenkins buildbot

http://darjeeling.iro.umontreal.ca:8080/job/Theano%20gpu/550/testReport/junit/theano.gpuarray.tests.test_dnn/TestDnnInferShapes/test_conv_time_on_shape_change_valid_conv/

If we keep it like that, then we will frequently have useless user questions. We will loose our own time and user time. If we bump the major version, I don't think it will give a good user error. We need to fix that. It could be for an 0.9rc2, but it should be before 0.9.